configs/daemonset-col-config.yaml

```diff
@@ -14,41 +12,6 @@ processors:
       - key: cluster
         value: ${env:KM_CLUSTER_NAME}
         action: insert
-  attributes/logs:
```

Comment:
This block should not be removed. They assign important details to log. In fact, add node.id attribute as well.